sheets/tasks.py

```diff
@@ -28,8 +28,10 @@ def handle_incomplete_coupon_assignments():
     Processes all as-yet-incomplete coupon assignment spreadsheets
     """
     coupon_assignment_handler = CouponAssignmentHandler()
-    spreadsheets = coupon_assignment_handler.process_assignment_spreadsheets()
-    return [(spreadsheet.id, spreadsheet.title) for spreadsheet in spreadsheets]
+    processed_spreadsheet_metadata = (
+        coupon_assignment_handler.process_assignment_spreadsheets()
+    )
+    return processed_spreadsheet_metadata
```

Comment:
Not important, but does this task actually need to return a result (and if not maybe `process_assignment_spreadsheets ` doesn't need to return anything either)?  I couldn't find any place where the results are used.

Reply:
The results are included in the logging output, which is potentially helpful for debugging

Example:
```
celery_1  | [2019-11-27 16:30:22,917: INFO/ForkPoolWorker-5] Task sheets.tasks.update_incomplete_assignment_delivery_statuses[18c957a7-b805-4369-b402-e651fe43907c] succeeded in 9.305771399987862s: [(27, 0), (34, 0), (37, 0), (40, 0), (41, 0)]
```